app/code/Magento/MediaGalleryCatalogUi/Ui/Component/Listing/Columns/InMenu.php

```diff
@@ -24,9 +24,9 @@ public function prepareDataSource(array $dataSource)
             $fieldName = $this->getData('name');
             foreach ($dataSource['data']['items'] as & $item) {
                 if (isset($item[$fieldName]) && $item[$fieldName] == 1) {
-                    $item[$fieldName] = 'Yes';
+                    $item[$fieldName] = '<span class="1">Yes</span>';
```

Comment:
@joweecaquicla is there a way to go without the HTML in the column value? (changing only the MFTF part)